concepticondata/conceptlists/Luangthongkum-2019-341.tsv

```diff
@@ -0,0 +1,342 @@
+Luangthongkum-2019-341-244	244	salt	1274	SALT
+Luangthongkum-2019-341-245	245	salty, sweet		
+Luangthongkum-2019-341-246	246	sand	671	SAND
+Luangthongkum-2019-341-247	247	scale (of fish)	266	SCALE
+Luangthongkum-2019-341-248	248	sell	1571	SELL
+Luangthongkum-2019-341-249	249	serow, mountain goat		
+Luangthongkum-2019-341-250	250	seven	1704	SEVEN
+Luangthongkum-2019-341-251	251	sew	1457	SEW
+Luangthongkum-2019-341-252	252	sharp	1396	SHARP
+Luangthongkum-2019-341-253	253	sheep	1331	SHEEP
+Luangthongkum-2019-341-254	254	shivering	52	SHIVER
+Luangthongkum-2019-341-255	255	shoot (v.)	1172	SHOOT
+Luangthongkum-2019-341-256	256	short, low	2484	SMALL (NOT TALL)
+Luangthongkum-2019-341-257	257	shoulder	1482	SHOULDER
+Luangthongkum-2019-341-258	258	shove (with the foot)		
+Luangthongkum-2019-341-259	259	six	1703	SIX
+Luangthongkum-2019-341-260	260	skink		
+Luangthongkum-2019-341-261	261	sleeping	1585	SLEEP
+Luangthongkum-2019-341-262	262	slippery	3702	SLIPPERY
+Luangthongkum-2019-341-263	263	smell (v.)	1586	SMELL (PERCEIVE)
+Luangthongkum-2019-341-264	264	snail (land~)	1543	SNAIL
+Luangthongkum-2019-341-265	265	snail (water~)		
+Luangthongkum-2019-341-266	266	snake	730	SNAKE
+Luangthongkum-2019-341-267	267	son-in-law	1056	SON-IN-LAW
+Luangthongkum-2019-341-268	268	sour	1906	SOUR
+Luangthongkum-2019-341-269	269	spider	843	SPIDER
+Luangthongkum-2019-341-270	270	spit (v.)	1440	SPIT
+Luangthongkum-2019-341-271	271	spleen	1744	SPLEEN
+Luangthongkum-2019-341-272	272	split (bamboo strip)		
+Luangthongkum-2019-341-273	273	spray (v.)		
+Luangthongkum-2019-341-274	274	spread out (a mat)	38	SPREAD OUT
+Luangthongkum-2019-341-275	275	squeeze, massage		
+Luangthongkum-2019-341-276	276	squirrel	1628	SQUIRREL
+Luangthongkum-2019-341-277	277	stand (v.)	1442	STAND
+Luangthongkum-2019-341-278	278	star	1430	STAR
+Luangthongkum-2019-341-279	279	steal (v.)	713	STEAL
+Luangthongkum-2019-341-280	280	stingy	1774	STINGY
+Luangthongkum-2019-341-281	281	stomach	819	STOMACH
+Luangthongkum-2019-341-282	282	stone, rock	2125	STONE OR ROCK
+Luangthongkum-2019-341-283	283	string, rope	1218	ROPE
+Luangthongkum-2019-341-284	284	suck	1421	SUCK
+Luangthongkum-2019-341-285	285	sun	1343	SUN
+Luangthongkum-2019-341-286	286	sunlight, hot		
+Luangthongkum-2019-341-287	287	sway, swing	573	SWAY
+Luangthongkum-2019-341-288	288	swollen, swelling		
+Luangthongkum-2019-341-289	289	tail	1220	TAIL
+Luangthongkum-2019-341-290	290	tasteless, faded	1771	TASTELESS
+Luangthongkum-2019-341-291	291	ten	1515	TEN
+Luangthongkum-2019-341-292	292	tendon	1137	TENDON
+Luangthongkum-2019-341-293	293	termite	883	TERMITE
+Luangthongkum-2019-341-294	294	termite (winged~)		
+Luangthongkum-2019-341-295	295	termite mound		
+Luangthongkum-2019-341-296	296	thick	1244	THICK
+Luangthongkum-2019-341-297	297	thin (as of cloth)	2306	THIN (OF LEAF AND CLOTH)
+Luangthongkum-2019-341-298	298	thousand	1843	THOUSAND
+Luangthongkum-2019-341-299	299	thread (a needle)	1161	THREAD
```

Comment:
It's not marked as a verb, but it sounds like one: to thread a needle? Since there is cotton thread mapped to THREAD already, I would unmap this one.